indico/legacy/webinterface/tpls/RoomBookingNewBookingSelectRoom.tpl

```diff
@@ -44,14 +43,25 @@
         % endif
 
         function initWidgets() {
-            $('#roomselector').roomselector({
-                allowEmpty: false,
-                rooms: rooms,
-                myRooms: myRooms,
-                roomMaxCapacity: maxRoomCapacity,
-                userData: ${ 'rbUserData' if not ignore_userdata else {}},
-                selectName: '${ form.room_ids.name }',
-                selectedRooms: ${ form.room_ids.data | j, n }
+            $.ajax({
+                url: '${ url_for('rooms_new.available_rooms') }',
+                method: 'GET',
+                dataType: 'json',
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function(data) {
+                    if (data.success) {
+                        $('#roomselector').roomselector({
+                            allowEmpty: false,
+                            rooms: JSON.parse(data.rooms),
+                            myRooms: JSON.parse(data.my_rooms),
+                            roomMaxCapacity: maxRoomCapacity,
+                            userData: ${ 'rbUserData' if not ignore_userdata else {}},
+                            selectName: '${ form.room_ids.name }',
+                            selectedRooms: ${ form.room_ids.data | j, n }
+                        });
+                    }
+                }
```

Comment:
I'm not sure this is needed. We don't need to have the old interface use the new endpoint. The old interface can still use the old way of doing things™️.

Reply:
Yeah, I forgot to remove it..